### PR TITLE
Revised 7.62x25 item description

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
I have revised the description of the 7.62x25 to establish the fact that it is the Type 51/Type P.

### CODE REVIEW

File: data/json/items/ammo.json [hunk 1]
```diff
@@ -2219,7 +2219,7 @@
         "name_plural" : "7.62mm Type P",
         "symbol" : "=",
         "color" : "light_gray",
-        "description" : "This small caliber pistol round offers good armor penetration at the cost of slightly less damage.  It is rarely used outside of the Chinese army.",
+        "description" : "A subsonic cartridge derived from the 7.62 Tokarev, designed for silenced firearms. It offers good armor penetration at the cost of slightly less damage.",
```
Comment (Rivet-the-Zombie) [inline data/json/items/ammo.json]: Double space after periods please.

**Comment by BorkBorkGoesTheCode** [comment]
Done.

**Comment by BorkBorkGoesTheCode** [comment]
Is there another problem @Rivet-the-Zombie ?

**Comment by BorkBorkGoesTheCode** [comment]
The description now differentiates between the 7.62x25mm and the 7.62x39mm.

**Comment by BorkBorkGoesTheCode** [comment]
I have added the second space.

**Pull request merged**